### DISCUSSION TITLE
Welcome to DexLib Discussions!

### DISCUSSION THREAD

**Discussion by mounibec** [paraswap/paraswap-dex-lib#301] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AvantgardeScott** [comment]
I have a question regarding the Commercial API's TOS.

If I am using your API on my website to facilitate aggregated swaps, do I have to Geoblock certain countries from accessing my website?

Furthermore:
Does the API endpoint have a built in geoblock preventing access from certain countries?
